guide/popular-topics/intents.md

```diff
@@ -0,0 +1,8 @@
+# Gateway Intents
+
+Gateway Intents were introduced to the library in v12 and allow you to pick-and-choose which events you wish your bot to recieve and/or omit.
+Intents are groups of pre-defined events which the discord.js client will conditionally subscribe to. For example, omiting the `GUILD_PRESENCES` intent would prevent the discord.js client from recieving any member-related presence events. Prior to this feature, the discord.js client would recieve all events, thus consuming memory for data you may not use.
```

Comment:
```suggestion
Intents are groups of pre-defined events which the discord.js client will conditionally subscribe to. For example, omiting the `GUILD_PRESENCES` intent would prevent the discord.js client from receiving any member-related presence events. Prior to this feature, the discord.js client would recieve all events, thus consuming memory for data you may not use.
```

Reply:
This is already covered by my review, and you missed the other typos anyway

Reply:
Woops, I overlooked that, I apologize.